modules/ROOT/pages/import-others.adoc

```diff
@@ -24,4 +24,7 @@ See link:https://neo4j.com/docs/getting-started/languages-guides/integration-too
 
 If your data is not in a CSV format, using APOC Load procedures allows you to ingest data from formats including json, xml and arrow.
 APOC is a library of user-defined procedures and functions that extends the use of Cypher.
-See link:https://neo4j.com/docs/apoc/current/overview/apoc.load/[APOC user guide -> apoc.load] for more information about the load procedure.
\ No newline at end of file
+See link:https://neo4j.com/docs/apoc/current/overview/apoc.load/[APOC user guide -> apoc.load] for more information about the load procedure.
```

Comment:
Can it say _Explore data importing options with the [Importing Data Fundamentals course on GraphAcademy](https://graphacademy.neo4j.com/courses/importing-fundamentals/?ref=docs-promo-import)._ instead? (Ie drop the _your_)

Reply:
https://github.com/neo4j-graphacademy/courses/pull/226